### ISSUE TITLE
Restart nach 1/2 Jahr: Keine Messwerte mehr

### ISSUE THREAD

**Issue by LukaGeba** [opendata-stuttgart/sensors-software#980]
Mein Sensor war seit ca. 6 Monaten nicht mehr aktiv.
Nun habe ich ihn wieder angeschlossen, auch ein FirmwareUpdate gemacht und ich habe trotzdem leider keine Messwerte mehr.
Weder PM noch Temp. Was kann ich tun?

<img width="237" alt="image" src="https://user-images.githubusercontent.com/123839599/215291084-d56f0682-5125-4159-862b-fe896882fc91.png">
<img width="343" alt="image" src="https://user-images.githubusercontent.com/123839599/215291127-4e3c4c35-5930-41bf-8887-8e6404040fc9.png">

**Comment by pjgueno** [comment]
Can you check the config. Are you sure it is the right sensors?
Oxidation on the pins ?
What about devices.Sensor.community ?
Please reflash with the firmware flasher.

**Comment by LukaGeba** [comment]
Yes, I have oxidation on the Temp.Sensor. But that wouldn't explain why the PM data isn't available either?

I flashed it with "latest.bin", Airrohr Flasher v0.3.4.
I also removed the flash file beforehand and flashed again.

Config: Sensor 1 is SDS011, Sensor 2: DHT22

I am not sure about the correct name / setting for my temp sensor.

**Comment by pjgueno** [comment]
Check the config on devices.sensor.community.

**Comment by pjgueno** [comment]
Send picture for the temp sensor.
